screenpipe-app-tauri/components/store/credit-purchase-dialog.tsx

```diff
@@ -51,7 +52,7 @@ export function CreditPurchaseDialog({
         </DialogHeader>
         <div className="space-y-4">
           <p className="text-sm text-muted-foreground">
-            you need {requiredCredits} credits but only have {currentCredits}
+            you need {requiredCredits} <Tooltip content="Credits are used to purchase services. Learn more about costs and details."><span className="underline cursor-pointer">credits</span></Tooltip> but only have {currentCredits}
```

Comment:
nice

is that supposed to be a link the learn more?

maybe could be a page in the docs